bears/js/ESLintBear.py

```diff
@@ -47,7 +47,12 @@ def create_arguments(filename, file, config_file,
 
     @staticmethod
     def generate_config(filename, file):
-        return '{"extends": "eslint:recommended"}'
+        return """
+        {
+            "extends": "eslint:recommended",
+            "parser": "babel-eslint",
```

Comment:
I think somebody raised a concern on wether we want this to be default or not but if you think it's safe I'd go with it. Still I'd like you to confirm that once again.